jobscheduler/scheduler-component-shared/src/java/org/sakaiproject/component/app/scheduler/SchedulerManagerImpl.java

```diff
@@ -745,4 +735,39 @@ public boolean isStartScheduler() {
    public void setStartScheduler(boolean startScheduler) {
        this.startScheduler = startScheduler;
    }
+
+    @Override
+    public void start() {
+        if (isStartScheduler()) {
+            try {
+                scheduler.start();
+            } catch (SchedulerException e) {
+                LOG.error("Failed to start the scheduler.", e);
+            }
+        } else {
+            LOG.info("Scheduler Not Started, startScheduler=false");
+        }
+    }
+
+    @Override
+    public void stop() {
+        try{
```

Comment:
Probably should make a call to super() as your overriding the application context stop().